src/Catty/DataModel/Project/Project.m

```diff
@@ -252,6 +252,31 @@ - (NSArray*)allObjectNames
     return [objectNames copy];
 }
 
+- (NSArray<SpriteObject*>*)allObjects
+{
+    NSMutableArray<SpriteObject*> *allObjects = [[NSMutableArray alloc] initWithArray: _objectList];
+    
+    for(NSUInteger index = 0; index < [self.userData.objectVariableList count]; index++) {
```

Comment:
Is there a reason why you did not directly add the `objectList` to the array? The `objectVariableList` of the `UserDataContainer` is deprecated and will be removed. Although the `objectList` is also deprecated, we should keep it as simple as possible for the moment.